index.html

```diff
@@ -269,111 +269,111 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
+                            src="./resources/assets/microsoft_logo.png"
```

Comment:
Please remove the logo from the file name.

Reply:
done